apps/src/clientApi.js

```diff
@@ -142,11 +161,21 @@ ApiRequestHelper.prototype.delete = function (localUrl, callback) {
 };
 
 /**
- * @param {!string} localUrl
+ * @param {!string} id
+ * @param {!Object} value
+ * @param {!NodeStyleCallback} callback
+ */
+ClientApiRequest.prototype.update = function (id, value, callback) {
+  this.post('/' + id, value, callback);
+};
+
+/**
+ * @param {!string|number} id Id to delete
```

Comment:
I don't know for sure that we don't, but I suspect we don't. Making it required is slightly riskier, but is cleaner if accurate. I think I'll make the change, and watch out for things breaking.